lib/model/CCategoryExamplesCollector.cc

```diff
@@ -143,6 +144,18 @@ void CCategoryExamplesCollector::clear() {
     m_ExamplesByCategory.clear();
 }
 
+void CCategoryExamplesCollector::debugMemoryUsage(core::CMemoryUsage::TMemoryUsagePtr mem) const {
+    mem->setName("CCategoryExamplesCollector");
+    core::CMemoryDebug::dynamicSize("m_ExamplesByCategory", m_ExamplesByCategory, mem);
+}
+
+//! Get the memory used by this examples collector.
```

Comment:
```suggestion
//! Return the memory used by this examples collector.
```

Reply:
This doxygen comment shouldn't have been in the `.cc` file at all.  I have removed it.